tests/difficulty_test.go

```diff
@@ -52,11 +52,8 @@ func TestDifficulty(t *testing.T) {
 
 	// files are 2 years old, contains strange values
 	dt.skipLoad("difficultyCustomHomestead\\.json")
-	dt.skipLoad("difficultyMorden\\.json")
-	dt.skipLoad("difficultyOlimpic\\.json")
```

Comment:
if remove `dt.skipLoad` -  then this tests will run. You probably mean opposite?

Reply:
I think actually it will skip them. Since we cannot remove them from the `tests` repo, this is one of the ways to disable them